crates/astria-bridge-withdrawer/src/bridge_withdrawer/startup.rs

```diff
@@ -0,0 +1,614 @@
+            }
+            res = startup_task => {
+                match res {
+                    Ok(Ok(())) => Ok(()),
+                    Ok(Err(err)) => {
+                        error!(%err, "startup failed");
+                        Err(err)},
+                    Err(reason) => {
+                        Err(reason.into())
+                    }
+                }
+            }
+        )
+    }
+
+    /// Confirms configuration values against the sequencer node. Values checked:
+    ///
+    /// - `self.sequencer_chain_id` matches the value returned from the sequencer node's genesis
+    /// - `self.fee_asset_id` is a valid fee asset on the sequencer node
+    /// - `self.sequencer_key.address` has a sufficient balance of `self.fee_asset_id`
+    ///
+    /// # Errors
+    ///
+    /// - `self.chain_id` does not match the value returned from the sequencer node
+    /// - `self.fee_asset_id` is not a valid fee asset on the sequencer node
+    /// - `self.sequencer_key.address` does not have a sufficient balance of `self.fee_asset_id`.
+    async fn confirm_sequencer_config(&mut self) -> eyre::Result<()> {
+        // confirm the sequencer chain id
+        let actual_chain_id =
+            get_sequencer_chain_id(self.sequencer_cometbft_client.clone(), self.state.clone())
+                .await
+                .wrap_err("failed to get chain id from sequencer")?;
+        ensure!(
+            self.sequencer_chain_id == actual_chain_id.to_string(),
+            "sequencer_chain_id provided in config does not match chain_id returned from sequencer"
+        );
+
+        // confirm that the fee asset ID is valid
+        let allowed_fee_asset_ids_resp =
+            get_allowed_fee_asset_ids(self.sequencer_cometbft_client.clone(), self.state.clone())
+                .await
+                .wrap_err("failed to get allowed fee asset ids from sequencer")?;
+        ensure!(
+            allowed_fee_asset_ids_resp
+                .fee_asset_ids
+                .contains(&self.expected_fee_asset_id),
+            "fee_asset_id provided in config is not a valid fee asset on the sequencer"
+        );
+
+        // confirm that the sequencer key has a sufficient balance of the fee asset
+        let fee_asset_balances = get_latest_balance(
+            self.sequencer_cometbft_client.clone(),
+            self.state.clone(),
+            self.address,
+        )
+        .await
+        .wrap_err("failed to get latest balance")?;
+        let fee_asset_balance = fee_asset_balances
+            .balances
+            .into_iter()
+            .find(|balance| balance.denom.id() == self.expected_fee_asset_id)
+            .ok_or_eyre("withdrawer's account does not have the minimum balance of the fee asset")?
+            .balance;
+        ensure!(
+            fee_asset_balance >= self.expected_min_fee_asset_balance,
+            "sequencer key does not have a sufficient balance of the fee asset"
```

Comment:
```suggestion
            "withdrawer account does not have a sufficient balance of the fee asset"
```

for consistency w above